src/searchApiClient.ts

```diff
@@ -0,0 +1,7 @@
+export function fetchSearchResultsFromAPI(searchTerm) {
+  // Note: this API seems a bit unstable and sometimes it takes a long time to respond, especially
+  // for broad searches
+  return fetch(
+    `https://cat-api-bjoerge.sanity-io1.now.sh/cats?query=${searchTerm}`
+  ).then((res) => res.json())
```

Comment:
We handle the happy path here, but we should add some error handling logic for the case where the API throws something other than a 200 response.